<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wenjian</class>
 <widget class="QWidget" name="wenjian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1387</width>
    <height>872</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1387</width>
    <height>872</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1387</width>
    <height>872</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="YaSuo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/resourse/wenjian6.jpg);
font: 18pt &quot;04b_21&quot;;</string>
   </property>
   <property name="text">
    <string>


压缩</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="JieYa">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>370</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/resourse/jieya9.png);
font: 18pt &quot;04b_21&quot;;</string>
   </property>
   <property name="text">
    <string>


解压</string>
   </property>
  </widget>
  <widget class="QPushButton" name="jishi">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>80</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/resourse/simple-green.jpg);
font: 28pt &quot;04b_21&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bianmaBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>压缩时生成编码文件</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="jindu">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>831</width>
     <height>51</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>230</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
